packages/telemetry/src/slog-to-otel.js

```diff
@@ -0,0 +1,640 @@
+            cause = {
+              name,
+              context: crankSpan.spanContext(),
+              attrs: {
+                target,
+                method,
+                result,
+                ...attrs,
+              },
+            };
+          }
+          crankNumToCause.init(slogAttrs.crankNum, cause);
+        }
+        break;
+      }
+      case 'deliver-result': {
+        const {
+          dr: [status, _1, meterResult],
+          // ...attrs
+        } = slogAttrs;
+        spans.get(getCrankKey()).setAttributes(
+          cleanAttrs({
+            status,
+            ...meterResult,
+          }),
+        );
+        break;
+      }
+      case 'syscall': {
+        const { ksc, vsc: _1, ...attrs } = slogAttrs;
+        if (!ksc) {
+          break;
+        }
+
+        const crankSpan = spans.get(getCrankKey());
+        /**
+         * @param {string} name
+         * @param {Record<string, any>} [additionalAttributes]
+         * @param {SpanOptions} [options ]
+         */
+        const makeSyscallSpan = (
+          name,
+          additionalAttributes = {},
+          options = {},
+        ) =>
+          spans.startNamed(
+            name,
+            [`syscall`, slogAttrs.crankNum],
+            crankSpan,
+            {
+              syscall: ksc[0],
+              ...attrs,
+              ...additionalAttributes,
+            },
+            options,
+          );
+        // TODO: get type from packages/SwingSet/src/kernel/kernelSyscall.js
+        switch (ksc[0]) {
+          case 'send': {
+            const [_tag, target, { method, result }] = ksc;
+            const name = `E(${target}).${method}`;
+            const links = [];
+            if (crankNumToCause.has(attrs.crankNum)) {
+              const parentCause = crankNumToCause.get(attrs.crankNum);
+              links.push({
+                attributes: { syscall: ksc[0] },
+                context: parentCause.context,
+              });
+            }
+            const syscall = makeSyscallSpan(
+              name,
+              {
+                target,
+                method,
+                result,
+              },
+              { links },
+            );
+            if (result) {
+              // Track call graph.
+              const cause = { name, context: syscall.spanContext() };
+              kernelPromiseToSendingCause.init(result, JSON.stringify(cause));
+            }
+            break;
+          }
+          case 'invoke': {
+            const [_, target, method] = ksc;
+            makeSyscallSpan(`D(${target}).${method}`, { target, method });
+            break;
+          }
+          case 'resolve':
+          case 'subscribe':
+          case 'vatstoreGet':
+          case 'vatstoreSet':
+          case 'vatstoreDelete':
+          case 'dropImports':
+          case 'retireImports':
+          case 'retireExports': {
+            // TODO: Maybe too noisy and mostly irrelevant?
```

Comment:
`resolve` seems important, but yeah we can probably ignore the rest for now. We need to remember that they were skipped when we're analyzing, though, else we're going to be concerned about all the GC that isn't happening, and confused about how the vat is managing virtual objects without vatstore calls.